packages/components/pie-thumbnail/src/thumbnail.scss

```
@@ -1 +1,28 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+// Base thumbnail styles
+.c-thumbnail {
+  --thumbnail-size-small: var(--dt-spacing-g);
+  --thumbnail-border-radius-small: var(--dt-radius-rounded-b);
+  --thumbnail-bg-color: var(--dt-color-container-default);
+  --thumbnail-border-outline: 1px solid var(--dt-color-border-default);
+
+  box-sizing: border-box;
+  width: var(--thumbnail-size-small);
+  height: var(--thumbnail-size-small);
+  border-radius: var(--thumbnail-border-radius-small);
+  border: var(--thumbnail-border);
```

`--thumbnail-border` isn't defined. I would define the border styles a bit differently so the border is applied in all variants, and the difference is the border color to maintain the same size. (if you go between variant stories, you will notice the page shifts) 

```suggestion
  define the following var and remove `--thumbnail-border-outline`
   --thumbnail-border-color: transparent
     
     border: 1px solid transparent
   
```

and 

```
 &.c-thumbnail--outline {
    --thumbnail-border-color:  var(--dt-color-border-default);
  }
```